plugin.js

```diff
@@ -120,19 +154,19 @@ const buildRouter = (admin, predefinedRouter) => {
  *   cookiePassword: 'somepassword',
  * }, [router])
 */
-const buildAuthenticatedRouter = (admin, auth, predefinedRouter, sessionOptions = {}) => {
+const buildAuthenticatedRouter = (admin, auth, predefinedRouter, options = {}) => {
   if (!session) {
     throw new Error(['In order to use authentication, you have to install',
       ' express-session package'].join(' '))
   }
   const router = predefinedRouter || express.Router()
   router.use(session({
-    ...sessionOptions,
+    ...options.sessionOptions,
```

Comment:
this can be undefined so I would change that to:

```
...(options.sessionOptions || {})
```

Reply:
When someone passes no options, it defaults to empty object. `sessionOptions` property will be undefined. When using spread operator on undefined it will add nothing, meaning it's safe to use spread operator on undefined in objects. I can fix this for readability if you want.